grails-app/domain/org/pih/warehouse/inventory/CycleCount.groovy

```diff
@@ -127,8 +127,11 @@ class CycleCount {
     /**
      * @return the cycle count item matching the given input (which uniquely identifies the item within the count).
      */
-    CycleCountItem getCycleCountItem(Location binLocation, InventoryItem inventoryItem, int countIndex) {
```

Comment:
from my comment on the other PR https://github.com/openboxes/openboxes/pull/5125/files#r1993935571

Replaces CycleCountService.findItemCreatedFromCustom